homeassistant/components/homekit_controller/button.py

```diff
@@ -84,7 +84,7 @@ def async_add_characteristic(char: Characteristic) -> bool:
                 entity.old_unique_id, entity.unique_id, Platform.BUTTON
             )
 
-        async_add_entities(entities, True)
+        async_add_entities(entities)
```

Comment:
this was a bug that triggered a ble connection at startup if there was a button entity